'async' is a reserved word in Python >= 3.7

Same issue and solution as CSAILVision/IBD#1

[flake8](http://flake8.pycqa.org) testing of https://github.com/CSAILVision/semantic-segmentation-pytorch on Python 3.7.1

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./lib/nn/parallel/data_parallel.py:16:31: E999 SyntaxError: invalid syntax
        v = obj.cuda(dev, async=True)
                              ^
./models/models.py:172:27: F821 undefined name 'UPerNetTmp'
            net_decoder = UPerNetTmp(
                          ^
1     E999 SyntaxError: invalid syntax
1     F821 undefined name 'UPerNetTmp'
2
```

--- a/lib/nn/parallel/data_parallel.py
+++ b/lib/nn/parallel/data_parallel.py
@@ -13,7 +13,7 @@ def async_copy_to(obj, dev, main_stream=None):
     if torch.is_tensor(obj):
         obj = Variable(obj)
     if isinstance(obj, Variable):
-        v = obj.cuda(dev, async=True)
+        v = obj.cuda(dev, non_blocking=True)
         if main_stream is not None:
             v.data.record_stream(main_stream)
         return v

